src/app/services/challenge.service.ts

```diff
@@ -90,6 +92,16 @@ export class ChallengeService {
     this.hostTeamSource.next(hostTeam);
   }
 
+
+  /**
+   * Update isHost Status.
+   * @param isHost boolean for isHost status.
+   */
+  changeHostStatus(isHost: any) {
```

Comment:
If you know the data type is boolean, then it is better to write it specifically rather than using `any`.